.env

```diff
@@ -0,0 +1,17 @@
+MOCHA_PASSWORD=Ape1sOn4Hill!
```

Comment:
secrets and passwords should not be checked in. need to invalidate and regenerate.